mpsl/doc/cx.rst

```diff
@@ -34,39 +49,43 @@ Each implementation exposes the same API to allow for an easier replacement that
 
 The radio coexistence implementation provided in nRF Connect SDK is the following:
 
+*nRF700x Coexistence*
+   This PTA interface is compatible with the nRF700x Coexistence interface. 
+
 *Thread Radio Coexistence*
-   This PTA interface is compatible with the 3-wire PTA interface defined in the *Thread Radio Coexistence recommendations* document available to Thread Group members.
+   This PTA interface is compatible with the generic 3-wire PTA interface defined in the *Thread Radio Coexistence recommendations* document available to Thread Group members.
```

Comment:
I based this change on the description of the mpsl_cx_thread.c file, which says "This file implements generic Coexistence interface according to Thread Radio Coexistence whitepaper.". The word `generic` seems to appear throughout the relevant documentation, therefore I thought it would fit here for consistency.